packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx

```diff
@@ -126,16 +126,18 @@ class CollapsibleMenuViewItem extends React.Component {
         />
       );
     } else if (isCollapsibleGroupItem) {
-      item = (
+      const button = (
         <ButtonGroup.Button
           {...attributes}
           icon={icon}
           text={faceUpText}
-          isReversed={isReversed}
-          isSelected={isSelected}
           isDisabled={isDisabled}
         />
       );
+
+      item = React.cloneElement(button, {
+        className: cx({ 'is-active': isSelected }),
+      });
```

Comment:
This will overwrite the original Button classNames, should they be merged?

Reply:
@benbcai Sync with @StephenEsser on this.

Reply:
Yes, the classNames should be merged. However, I should not need to set the `is-active` class here to indicate that the button is selected. This is already done in ButtonGroup. However, when there are other pass through classNames in addition to `is-active`, the `is-active` className does not get binded to ButtonGroupButton as intended because it is being set in ButtonGroup.

Got some help from @StephenEsser in suggesting to use [composes](https://github.com/css-modules/css-modules#composing-from-other-files). Basically `is-active` is a className in ButtonGroupButton but it is ButtonGroup that manages the selection and sets the `is-active` class on ButtonGroupButton. Since the styles are modular, ButtonGroup would need to compose the `is-active` className from ButtonGroupButton.